### ISSUE TITLE
Tests im Hinblick auf Standort "Dezentrale Fachbibliotheken"

### ISSUE THREAD

**Issue by florianruckelshausen** [florianruckelshausen/cbs2folio-transformations#5]
- [ ] Sammler UB-Standort Unbekannt testen (mit Fantasiesaignatur)
- [ ] Magazin ZNL mit MAG 002 als Signaturanfang soll zu Standort ZNL-Magazin
- [ ] Aufsatzkatalogisat mit BIK=950 (PPN=173530834) soll zu Standort "Aufsatzkatalogisate"

**Comment by michaelfreiberg** [comment]
Frage: Soll der Quart-Format-Bereich `**4o** 49.000.00 - **4o** 49.999.99` so wie `49.000.00 bis 49.999.99` auch dem Keller-Magazin zugewiesen werden?

**Comment by michaelfreiberg** [comment]
Frage: Sollten wir statt `mag bap 1`  nicht besser den Präfix `mag bap` definieren, da vermutlich alle BAp-Signaturen mit MAG-Vorsatz im Magazin 3. OG aufgestellt werden?

**Comment by michaelfreiberg** [comment]
TODO: Wir bräuchten noch einen Standort für die Dummys.

**Comment by florianruckelshausen** [comment]
> Frage: Soll der Quart-Format-Bereich `4o 49.000.00 - 4o 49.999.99` so wie `49.000.00 bis 49.999.99` auch dem Keller-Magazin zugewiesen werden? Aktuell Standort = ILN204/CG/ZNL/Mag.

Ja, nimm Keller (ist auch so im LBS).

**Comment by florianruckelshausen** [comment]
> Frage: Sollten wir statt `mag bap 1` nicht besser den Präfix `mag bap` definieren, da vermutlich alle BAp-Signaturen mit MAG-Vorsatz im Magazin 3. OG aufgestellt werden?

Ja, können wir machen. Die Bap Bestände, die nach Pohlheim gezogen sind, haben Bap als Signatur (ohne MAG-Vorsatz).

**Comment by florianruckelshausen** [comment]
> Rückfrage:
> 
> > Artikel / Aufsätze aus ZS / Sammelbänden, erkennbar an Kategorie 0500 Position 2 ist ein "o" (PPN=123639743)
> 
> Auch dem Standort 950 zuweisen?

Ja. Die beiden Anforderungen überschneiden sich ein bisschen. Bei den Aufsatzkatalogisaten bin ich unsicher, was es da alles für Varianten gibt. Aber sie sollten zu Standort-UUID 3522ba70-ee87-4a26-a854-469dff54723a ("Aufsatzkatalogisate und übergeordnete Aufnahmen").

**Comment by florianruckelshausen** [comment]
> TODO: Wir bräuchten noch einen Standort für die Dummys.

Auch die sollen zu Standort-UUID 3522ba70-ee87-4a26-a854-469dff54723a ("Aufsatzkatalogisate und übergeordnete Aufnahmen"). Ich würde da erstmal nicht weiter differenzieren und alle Aufsätze, c-Sätze (übergeordnete Schriftenreihen) und d-Sätze (übergeordnete Mehrbänderaufnahmen) da rein mappen.
